export/famplex_groundings.tsv

```diff
@@ -1317,9 +1334,15 @@ Luteinizing hormone	LH	fplx	Family
 Ly49D	Q60651	uniprot	Gene_or_gene_product
 MAC	MAC	fplx	Family
 MAC-1	MAC_1	fplx	Family
+MAP	MAP	fplx	Family
```

Comment:
Right, so the appearance of this synonym here that makes its way into Reach could be problematic (see comment above).